classic-sdk/core/src/main/java/com/microsoft/applicationinsights/TelemetryClient.java

```diff
@@ -58,20 +59,16 @@ public void trackEvent(
       @Nullable Map<String, String> properties,
       @Nullable Map<String, Double> metrics) {
 
-    if (isDisabled()) {
```

Comment:
it seems isDisabled() check is removed everywhere. Should we remove that boolean method too? It's only used in this class.

Reply:
it was _kind_ of needed, because we used it in the bytecode instrumentation, but I agree that it's confusing, so I removed it, and if we need that feature in the future (to disable 2.x telemetry), it should be a json configuration option anyways